core/src/main/java/org/mskcc/cbio/portal/servlet/QueryBuilder.java

```diff
@@ -274,7 +275,13 @@ protected void doPost(HttpServletRequest httpServletRequest,
             xdebug.logMsg(this, "Got Database Exception:  " + e.getMessage());
             forwardToErrorPage(httpServletRequest, httpServletResponse,
                                DB_CONNECT_ERROR, xdebug);
-        } catch (ProtocolException e) {
+        } catch (DbVersionException e) {
+            String errorMessage = "Mismatch between DB version and portal version. " + e.getMessage();
```

Comment:
There are at least three versions involved. The portal release version is displayed at the bottom of the portal footer ... the current public portal reports "cBioPortal Version 1.3.2-SNAPSHOT". This comes from pom.xml.

This is completely independent of the version numbers of the database schemas. To keep this clear, I think we should skew the version numbering now so that the numbers are not so easy to confuse.

There is a version number for the database stored in the 'info' table. For the public portal, the current "database version" is stored in a field called DB_SCHEMA_VERSION. The current value there is "1.3.0".

The warfile embeds properties which set an expected database version number.  The currently running war file for the public tomcat sets "db.version=1.3.0" in portal.properties. This value is compared against the string in the connected database. In this case, the expected database version number is 1.3.0, and the actual datbase version number matches, so there is no error reported. But neither of these match the "portal version".

I recommend we change the property name in portal.properties to "db.expected_schema_version". Some databases are updated with new data nightly or more frequently. We do not increment the version number of the database on data updates ... only schema updates, so I recommend that we talk in terms of db schema versions rather than db versions.

For this PR though, it is sufficient to change this message to not refer to the 'portal version'. maybe "DB Version Exception : " + e.getMessage();